src/services/completions.ts

```diff
@@ -2060,16 +2060,18 @@ namespace ts.Completions {
                 case KeywordCompletionFilters.None:
                     return false;
                 case KeywordCompletionFilters.All:
-                    return kind === SyntaxKind.AsyncKeyword || SyntaxKind.AwaitKeyword || !isContextualKeyword(kind) && !isClassMemberCompletionKeyword(kind) || kind === SyntaxKind.DeclareKeyword || kind === SyntaxKind.ModuleKeyword

```

Comment:
@andrewbranch Here! `SyntaxKind.AwaitKeyword` isn't compared `===kind` and since it's truthy it caused `.All` to let *all* keywords through.